docs/learn/benefits-lukso-standards.md

```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
 | Upgradeable Security           | ✅                         | ✅                          | ✅                                      | ✅ Through [LSP14](../standards/access-control/lsp14-ownable-2-step.md)                             |
 | Pre and Post Execution Hooks   | ✅                         | ✅                          | ✅                                      | ✅ Through [LSP20](../standards/accounts/lsp20-call-verification.md)                                |
 
-As shown from the table above, the [Universal Profile Browser Extension](/install-up-browser-extension) provides more features and a better user experience when interacting with dApps. Universal Profiles (known as 🆙) are smart contract accounts that offer many benefits.
+| Feature                                        | Benefits                                                                                                                                                                                                                                                                                                                           |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **🗄️ Metadata in one place**                   | Finished re-filling the same information again and again on each new app you register to!<br/>Your information is stored and fetched from the same place (your 🆙 storage) and displayed the same across all the dApps ecosystem!                                                                                                  |
+| **🎨 Customizable metadata**                   | Make your Universal Profile stand out from others! Give it a profile and cover image, and add as many amount of information with its unlimited storage.                                                                                                                                                                            |
```

Comment:
Is the storage really unlimited? 🤔

Reply:
> Is the storage really unlimited? 🤔

Yes it is practically unlimited because the data keys are `bytes32` which makes a very big range `2e256`

Reply:
Ah, okay. I was thinking rather about the `bytes` part for the values. How much can we upload and store? I guess it's limited by the 1) block's gas limit of 42m and 2) physical storage.

Reply:
Good question, I am actually not sure. A burnt pix has a very large amount of metadata in its `LSP4Metadata`, but we haven't tested the boundaries yet for a single `setData(bytes32,bytes)` transaction. 
Yes technically, it is limited to the block's gas limit for setting.

See the value under the `LSP4Metadata` here :)
https://erc725-inspect.lukso.tech/inspector?network=mainnet&address=0x3983151e0442906000dab83c8b1cf3f2d2535f82

Reply:
Thanks!